JSDemos/Demos/Toolbar/Multiline/React/styles.css

```diff
@@ -0,0 +1,64 @@
+.dx-resizable-handle::after {
+  content: "";
+  background-color: #337ab7;
+  position: absolute;
+  width: 12px;
+  height: 12px;
+  border: 1px solid #337ab7;
+  top: calc(50% - 7px);
+}
+
+.dx-resizable-handle-right::after {
+  right: -7px;
+}
+
+.dx-toolbar.dx-toolbar-multiline .dx-toolbar-item {
+  margin-bottom: 5px;
+}
+
+.dx-popup-content > .dx-template-wrapper.custom-color-picker {
+  width: 82px;
+  padding: 5px;
+}
+
+.resizable-container {
+  width: calc(100% - 6px);
+  padding: 10px;
+  height: 300px;
+  border: 1px dashed #dbdbdb;
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+.options-container {
+  margin-top: 20px;
+  padding: 20px;
+  background-color: rgba(191, 191, 191, 0.15);
+  position: relative;
+}
+
+.caption {
+  font-size: 18px;
+  font-weight: 500;
+  margin-bottom: 10px;
+}
+
+.separator {
+  height: 36px;
+  margin: 0 5px;
+  border-left: 1px solid #ddd;
+}
+
+.menu-separator {
+  height: 1px;
+  border-bottom: 1px solid #ddd;
+}
+
+.dx-toolbar-menu-section .dx-toolbar-menu-custom .dx-list-item-content:first-child > .menu-separator {
+  display: none;
+}
+
+.color {
```

Comment:
Seems you forgot to remove it...

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/c8c3b387ef97137bad2b5858c6be85cfff992081